### DISCUSSION TITLE
reviewers or assignees not working

### DISCUSSION THREAD

**Discussion by codewode** [renovatebot/renovate#13594] [Request Help]
The documentation is confusing, there are reviewers and assignees. Tried both with username, email. but it's not showing up. Here is my latest config. 

```
{
  "platform":"azure",
  "baseBranches":[
    "main"
  ],
  "reviewers":[
    "xxx@xxx.xx"
  ],
  "packageRules":[
    {
      "matchUpdateTypes":[
        "minor",
        "patch",
        "pin",
        "digest"
      ],
      "matchPackagePatterns":[
        "^(microsoft|Microsoft)"
      ],
      "groupName":"microsoft packages"
    }
  ]
}
```
what am I doing wrong?

I have cross checked, username is correct in the platform.

**Comment by rarkins** [comment]
They are only added at time of PR creation. So when you try different things, are you making sure that you are creating a new PR each time?

**Reply by codewode** [reply]
so the newly created PR is still missing reviewers/assignees, but your reply answered a different question about the "label", label is at least assigned for newly created PR. I will use the "automatically include reviewer" feature from the platform.
